pages/stats/StatsHandler.php

```diff
@@ -27,9 +33,109 @@ class StatsHandler extends PKPStatsHandler
+            'contextIds' => [$context->getId()],
+            'count' => $count,
+            'dateStart' => $dateStart,
+            'dateEnd' => $dateEnd,
+        ]);
+
+        $statsComponent = new \APP\components\StatsIssuePage(
+            $dispatcher->url($request, PKPApplication::ROUTE_API, $context->getPath(), 'stats/issues'),
+            [
+                'timeline' => $timeline,
+                'timelineInterval' => PKPStatisticsHelper::STATISTICS_DIMENSION_DAY,
+                'timelineType' => 'toc',
+                'tableColumns' => [
+                    [
+                        'name' => 'title',
+                        'label' => __('editor.issues.issueIdentification'),
+                    ],
+                    [
+                        'name' => 'tocViews',
+                        'label' => __('stats.issueTOCViews'),
+                        'value' => 'tocViews',
+                    ],
+                    [
+                        'name' => 'issueGalleyViews',
+                        'label' => __('stats.issueGalleyViews'),
+                        'value' => 'issueGalleyViews',
+                    ],
+                    [
+                        'name' => 'total',
+                        'label' => __('stats.total'),
+                        'value' => 'totalViews',
+                        'orderBy' => 'totalViews',
+                        'initialOrderDirection' => true,
+                    ],
+                ],
+                'count' => $count,
+                'dateStart' => $dateStart,
+                'dateEnd' => $dateEnd,
+                'dateRangeOptions' => [
+                    [
+                        'dateStart' => $dateStart,
+                        'dateEnd' => $dateEnd,
+                        'label' => __('stats.dateRange.last30Days'),
+                    ],
+                    [
+                        'dateStart' => date('Y-m-d', strtotime('-91 days')),
+                        'dateEnd' => $dateEnd,
+                        'label' => __('stats.dateRange.last90Days'),
+                    ],
+                    [
+                        'dateStart' => date('Y-m-d', strtotime('-12 months')),
+                        'dateEnd' => $dateEnd,
+                        'label' => __('stats.dateRange.last12Months'),
+                    ],
+                    [
+                        'dateStart' => '',
+                        'dateEnd' => '',
+                        'label' => __('stats.dateRange.allDates'),
+                    ],
+                ],
+                'orderBy' => 'total',
+                'orderDirection' => true,
+            ]
+        );
+        $templateMgr->setState($statsComponent->getConfig());
+        $templateMgr->assign([
+            'pageComponent' => 'StatsIssuesPage',
+            'pageTitle' => __('stats.issueStats'),
+            'pageWidth' => TemplateManager::PAGE_WIDTH_WIDE
```

Comment:
This doesn't work because a `max-width` property on `.app__page` is overriding it. I think we can safely remove the `max_width` property from `.app__page {` in `Page.vue`. Then the page will be wider.